etc/kayobe/kolla/config/grafana/dashboards/openstack/grafana_cloud_dashboard.json

```diff
@@ -836,7 +843,7 @@
         "current": {
           "selected": false,
           "text": "Prometheus",
-          "value": "Prometheus"
```

Comment:
I think we want to preserve the original line here. Please see this for context:

https://github.com/stackhpc/stackhpc-kayobe-config/commit/9dcc4ec836eefa4f2f797a31175cc860248a7107